packages/twenty-front/src/utils/string/formatDateString.ts

```diff
@@ -1,23 +1,36 @@
 import { DateFormat } from '@/localization/constants/DateFormat';
+import { formatDateISOStringToCustomUnicode35Format } from '@/localization/utils/formatDateISOStringToCustomUnicode35Format';
 import { formatDateISOStringToDate } from '@/localization/utils/formatDateISOStringToDate';
 import { formatDateISOStringToRelativeDate } from '@/localization/utils/formatDateISOStringToRelativeDate';
+import { FieldDateMetadataSettings } from '@/object-record/record-field/types/FieldMetadata';
 
 export const formatDateString = ({
   value,
   timeZone,
   dateFormat,
-  displayAsRelativeDate,
+  dateFieldSettings,
 }: {
   timeZone: string;
   dateFormat: DateFormat;
   value?: string | null;
-  displayAsRelativeDate?: boolean;
-}) => {
-  const formattedDate = value
-    ? displayAsRelativeDate
-      ? formatDateISOStringToRelativeDate(value)
-      : formatDateISOStringToDate(value, timeZone, dateFormat)
-    : '';
+  dateFieldSettings?: FieldDateMetadataSettings;
+}): string => {
+  if (!value) {
+    return '';
+  }
 
-  return formattedDate;
+  switch (dateFieldSettings?.displayFormat) {
+    case 'RELATIVE':
+      return formatDateISOStringToRelativeDate(value);
+    case 'USER_SETTINGS':
+      return formatDateISOStringToDate(value, timeZone, dateFormat);
+    case 'CUSTOM':
+      return formatDateISOStringToCustomUnicode35Format(
```

Comment:
Seems like "Unicode35" isn't a popular naming convention and we should just call all fields/function "unicode"